examples/medical-center/persist/medical_center.bal

```diff
@@ -0,0 +1,17 @@
+import ballerina/time;
```

Comment:
```suggestion
import ballerina/time;
import ballerina/persist as _;
```

Reply:
Fixed in [90235d8](https://github.com/ballerina-platform/module-ballerina-persist/pull/103/commits/90235d817df86c78d05ff4528baba5a18d2a584a)